cmd/openshift-install/create.go

```diff
@@ -427,8 +427,8 @@ func logComplete(directory, consoleURL string) error {
 		return err
 	}
 	logrus.Info("Install complete!")
-	logrus.Infof("Run 'export KUBECONFIG=%s' to manage the cluster with 'oc', the OpenShift CLI.", kubeconfig)
-	logrus.Infof("The cluster is ready when 'oc login -u kubeadmin -p %s' succeeds (wait a few minutes).", pw)
+	logrus.Infof("To access the cluster as the system:admin user, run 'export KUBECONFIG=%s'", kubeconfig)
+	logrus.Info("Manage the cluster with 'oc', the OpenShift CLI.  Run 'oc --help' for the menu.")
```

Comment:
Do we want this level of detail?  If folks need to be reminded about `--help`, I'd much rather be linking them to external docs for using `oc`.

Reply:
lol yup, good point.